AllReadyApp/Web-App/AllReady/wwwroot/css/mainPage.css

```diff
@@ -1,115 +0,0 @@
-﻿body {
-
-}
-
-/* Main page CSS */
-.home-banner {
-    background-color: #d58033;
-}
-
-.search-banner {
-    background-color: #d58033;
-    padding-bottom:55px;
-}
-
-input.main-search {
-    width: 400px;
-    display: inline-block;
-    border: 0;
-    font-size: 18px;
-    height: 50px;
-    padding: 0px 5px;
-    /*padding-left: 5px;
-    padding-right: 5px;
-    padding-top: 11px;
-    padding-bottom: 16px;*/
-}
-.main-search-button {
-    margin: 0;
-    cursor: pointer;
-}
-.main-search-button > img {
-    vertical-align: top;
-}
-::-webkit-input-placeholder,
-:-moz-placeholder,
-::-moz-placeholder,
-:-ms-input-placeholder
- {
-   color: #999999;
-}
-
-.sub-title {
-    margin-bottom: 50px;
-    color: #FFFFFF;
-}
-
-.main-heading {
-    color: #fff;
-}
-.event-results > h3 {
-    color: #fff;
-}
-.event-results a {
-    text-decoration: none;
-    font-size: 16px;
-}
-.event-results a:hover {
-    text-decoration: underline;
-}
-.event-results h3 {
-    color: #333333;
-    padding-top:20px;
-    padding-bottom:10px;
-}
-.col-md-12 h4 {
-    margin-top:50px;
-}
-.main-page-footer {
-    color:#ab6627;
-    font-size:small;
-    width:100%;
-    margin-top:50px;
-    margin-bottom: 20px;
-    height:100px;    
-}
-
-.navbar {
-    margin-bottom:0px;
-    background-color:#ab6627;
-    border-radius:0px;
-}
-
-.row {
-    margin-right:auto;
-    margin-left: auto;
-}
-/* End main page CSS */
-
-.inline-form {
-    display: inline-block;
-}
-
-.featured-campaign {
-    border: 1px solid #d58033;
-    padding: 6px;
-    margin: 10px;
-    background: #e4cdb8;
-}
```

Comment:
The `featured-campaign` styles are still used in `Home/Index.cshtml`. They should either move to `site.css` or to a `styles` section within `Home/Index.cshtml`. See `404.cshtml` for an example.